docs/asyncapi.yaml

```diff
@@ -86,61 +85,59 @@ components:
 
     crn:
       type: number
-      description: Customer reference Number
+      description: Customer Reference Number (consists of 10 digits)
       minimum: 1050000000
       maximum: 9999999999
       example: 1234567890
 
     sbi:
       type: number
-      description: Single Business Identifier
+      description: Single Business Identifier (consists of 9 digits)
       minimum: 105000000
       maximum: 999999999
       example: 123456789
 
     sourceSystem:
       type: string
-      description: The name of the source system that originated the message
+      description: Name of the source system that originated the message
       example: "AHWP"
       minlength: 3
       maxlength: 100
       pattern: "^[a-z0-9-_]+$"
 
     notifyTemplateId:
       type: string
-      description: Notification template identifier
+      description: Notify template identifier
       example: "f33517ff-2a88-4f6e-b855-c550268ce08a"
       format: uuid
 
     commsType:
       type: string
-      description: Type of communication (e.g., email, SMS)
+      description: Type of communication (e.g. email, SMS)
       default: "email"
       example: "email"
       enum: ["email"]
 
-    commsAddress:
+    commsAddresses:
       oneOf:
         - type: string
-          description: The user's contact email address
+          title: Accepts a single string
+          description: A single contact email address corresponding to one recipient
           example: "mrMegaFarmer@strawberryFortFarm.co.uk"
           minLength: 1
           maxLength: 320
           pattern: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}(\\.[a-zA-Z]{2,})?$"
-          title: accepts a single string
         - type: array
-          title: accepts an array of strings
-          items:
```

Comment:
Need to keep this items in here. It defines the schema for each item inside the array so won't work as intended if it's on the outside.